tests/components/mikrotik/__init__.py

```diff
@@ -0,0 +1,133 @@
+"""Tests for the Mikrotik component."""
+from homeassistant.components import mikrotik
+
+MOCK_DATA = {
+    mikrotik.CONF_NAME: "Mikrotik",
+    mikrotik.CONF_HOST: "0.0.0.0",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible binding to all interfaces.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4918093)